tests/genomics_tests/test_genomic_file_ingester.py

```diff
@@ -116,6 +116,44 @@ def test_replating_copy(self):
         self.assertEqual(copy_member.blockResearchReason, block_research_reason)
         self.assertEqual(copy_member.blockResearch, 1)
 
+    def test_validate_filenames(self):
+        job_controller = GenomicJobController(job_id=GenomicJob.AW1F_MANIFEST)
 
+        file_validator = GenomicFileValidator(
+            job_id=job_controller.job_id,
+            controller=job_controller
+        )
 
-
+        aw1f_files = ['UW_AoU_GEN_PKG-1234-567890_FAILURE.csv',
+                      'UW_AoU_GEN_PKG-1234-567890_FAILURE_v2.csv',
+                      'UW_AoU_GEN_PKG-1234-567890_FAILURE-v2.csv',
+                      'UW_AoU_GEN_PKG-1234-567890_FAILURE.v2.csv']
+        aw2_files = ['AoU_GEM_A2_manifest_2020-07-11.csv',
+                     'AoU_GEM_A2_manifest_2020-07-11_v2.csv']
+        aw4_arr_files = ['AoU_DRCB_GEN_2020-07-11-00-00-00.csv',
+                         'AoU_DRCB_GEN_2020-07-11-00-00-00_v2.csv']
+        aw4_wgs_files = ['AoU_DRCB_SEQ_2020-07-11-00-00-00.csv',
+                         'AoU_DRCB_SEQ_2020-07-11-00-00-00_v2.csv']
+
```

Comment:
While this works with the `assertEqual` method, I think it might be better to structure the tests a little differently. 
Maybe we could do something like looping thru the designated short-read job and lean into the targeted testing boolean methods like `assertTrue`/`assertFalse` and pass in the filename str as an argument.
Ex: 
```
for job_id in short_read_job_ids:
        file_validator.job_id = job_id
        self.assertTrue(file_validator.validate_filename(<filename_string>) // if true
        self.assertTrue(file_validator.validate_filename(<filename_string>) // if false
```

Reply:
Updated to use targeted testing boolean methods!